src/Components/Body.jsx

```diff
@@ -1,9 +1,35 @@
+import { Route, Routes } from "react-router-dom";
 import Reviews from "./Reviews";
+import ReviewSingle from "./ReviewSingle";
+import { useState, useEffect } from "react";
+import { getReviews, getReviewID, getCategories } from "../api";
+import { useParams } from "react-router-dom";
+
+export default function Body() {
+  const [reviews, setReviews] = useState([]);
+  // const { item_id } = useParams();
+  // const [singleReview, setSingleReview] = useState([]);
+
+  useEffect(() => {
+    // setIsLoading(true);
+    getReviews().then((data) => {
+      setReviews(data);
+      // setIsLoading(false);
+    });
+  }, []);
+
+  // useEffect(() => {
+  //   getReviewID(item_id).then((data) => {
+  //     setSingleReview(data);
+  //   });
+  // }, [item_id]);
 
-export default function Body({ reviews }) {
   return (
     <div className="Body mb-10 mt-10 p-5">
-      <Reviews reviews={reviews} />
+      <Routes>
+        <Route path="/reviews" element={<Reviews reviews={reviews} />} />
```

Comment:
your homepage is empty and there's no navigation to /reviews for the user, can you add a basic link for now to the reviews or render the reviews on the homepage, whichever you'd prefer